packages/features/vault/src/components/vault-sub-nav/use-vault-sub-nav.tsx

```diff
@@ -0,0 +1,38 @@
+import { animateScroll as scroll } from 'react-scroll';
+import { VAULT_SUB_NAV_ACTIONS } from '@notional-finance/shared-config';
+
+export const useVaultSubNav = () => {
+  const subNavData = [
+    {
+      label: 'Overview',
+      key: VAULT_SUB_NAV_ACTIONS.OVERVIEW,
+      anchor: VAULT_SUB_NAV_ACTIONS.OVERVIEW,
+    },
+    {
+      label: 'Market Returns',
+      key: VAULT_SUB_NAV_ACTIONS.MARKET_RETURNS,
+      anchor: VAULT_SUB_NAV_ACTIONS.MARKET_RETURNS,
+    },
+    // {
+    //   label: 'Yield Drivers',
+    //   key: VAULT_SUB_NAV_ACTIONS.YIELD_DRIVERS,
+    //   anchor: VAULT_SUB_NAV_ACTIONS.YIELD_DRIVERS,
+    // },
+    {
+      label: 'Strategy Details',
+      key: VAULT_SUB_NAV_ACTIONS.STRATEGY_DETAILS,
+      anchor: VAULT_SUB_NAV_ACTIONS.STRATEGY_DETAILS,
+    },
+    {
+      label: 'Back to top',
+      key: VAULT_SUB_NAV_ACTIONS.BACK_TO_TOP,
+      callback: () => scroll.scrollToTop(),
+    },
+    {
+      label: 'Full Documentation',
+      key: VAULT_SUB_NAV_ACTIONS.FULL_DOCUMENTATION,
+      href: 'https://docs.notional.finance/leveraged-vaults/leveraged-vaults/balancer-aura-wsteth-weth-strategy',
```

Comment:
This is likely going to have to change to a prop that gets passed in later, I guess we need to figure out how to get the right link in here...we can leave it as is for now